.github/workflows/refresh_deployment.yaml

```diff
@@ -0,0 +1,64 @@
+name: Refresh EC2 FINOS Waltz deployment
+
+on:
+  workflow_dispatch:
+  #schedule:
+  # Runs everyday at 00:00. (see https://crontab.guru)
+  #- cron: '0 0 * * *'
```

Comment:
I wonder why the schedule is commented out.

Reply:
No reason besides it activating automatically before we have a chance to run it manually and make sure everything works ok.

Reply:
I've removed the comments.